packages/core/src/dialects/abstract/query-generator.js

```diff
@@ -98,6 +98,10 @@ export class AbstractQueryGenerator extends AbstractQueryGeneratorTypeScript {
     throw new Error(`Schemas are not supported in ${this.dialect.name}.`);
   }
 
+  createTableQuery() {
+    throw new Error(`Create Table is not implemented in ${this.dialect.name}.`);
+  }
+
```

Comment:
```suggestion
```

It's implemented in all dialects. We only need this once we have migrated it to TypeScript